Expensify/App
# Revert "Merge pull request #6738 from azimgd/fix-reverted-list"

This reverts commit 1c80150632c684174270c019b7439854318471a4, reversing
changes made to 7b0752d5f973003ae7b51f8c466513626b3619c6.

cc @azimgd 

### Details
This inverted flatlist fix broke iOS and Android views with a few amount of messages

### Fixed Issues
$ https://github.com/Expensify/App/issues/6792

### Tests
1. Launch the app
1. Log in
1. Tap on Fab menu and Tap on New Room
1. Create a Room
1. Send some messages
1. _Verify_ they appear at the bottom

### QA Steps
1. Launch the app
1. Log in
1. Tap on Fab menu and Tap on New Room
1. Create a Room
1. Send some messages
1. _Verify_ they appear at the bottom

### Tested On


### Screenshots
![image](https://user-images.githubusercontent.com/22447860/146294180-f5478e6f-ce46-483e-9e19-4277295abb9a.png)
